NuGet.config

```diff
@@ -6,6 +6,7 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
+    <add key="msbuild17.7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-DotNet-msbuild-Trusted-5785ed5c/nuget/v3/index.json" />
```

Comment:
Is this needed for the api validation?
Just curious why we haven't need anything similar before.

Reply:
Yes. We don't need this once the package is on one of the other feeds--and when the package is published to nuget.org it will automatically go to `dotnet-public`. In the past we've waited for that. I wanted to experiment with using the `.0` build though, which is available only in that build-specific feed, because we said it was a final release version but haven't pushed it publicly yet.